CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CHANGELOG
 
+### v0.5.0
+- Adding @nypl/dgx-svg-icons package and updating styles.
```

Comment:
You also updated the title of the social media links for accessibility. That is worthy putting in!

Reply:
got it